### PR TITLE
Fix tests for forked module

### PR DESCRIPTION
**Pull request by tractorcow**
Use config API better

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,4 +29,4 @@ before_script:
   - cd ~/builds/ss
 
 script:
-  - phpunit silverstripe-lumberjack/tests/
\ No newline at end of file
+  - vendor/bin/phpunit lumberjack/tests/
```
Comment (micmania1) [inline .travis.yml]: I'm pretty sure this will still need to be 'silverstripe-lumberjack'

Comment (tractorcow) [inline .travis.yml]: I'd changed the vendor/projectname on this fork, which is what generates the actual install path.

Comment (micmania1) [inline .travis.yml]: There is benefit in keeping it prefixed with silverstripe-. You can just add silverstripe-* to your .gitignore which means people don't keep forgetting to add modules. It was a workaround as we don't use vendor/ or have a dedicated module folder.

Comment (tractorcow) [inline .travis.yml]: Ah I see, I didn't realise that. :)

**Pull request merged**